src/data/events/EventsD2ApiRepository.ts

```diff
@@ -81,34 +82,36 @@ export class EventsD2ApiRepository implements EventsRepository {
         const orgUnits = cleanOrgUnitPaths(orgUnitPaths);
 
         const fetchApi = async (orgUnit: string, page: number) => {
-            return this.api.events
+            return this.api.tracker.events
                 .get({
                     pageSize: 250,
                     totalPages: true,
                     page,
                     orgUnit,
-                    startDate: period !== "ALL" ? startDate.format("YYYY-MM-DD") : undefined,
-                    endDate: period !== "ALL" ? endDate.format("YYYY-MM-DD") : undefined,
-                    lastUpdated: lastUpdated ? moment(lastUpdated).format("YYYY-MM-DD") : undefined,
+                    occurredAfter: period !== "ALL" ? startDate.format("YYYY-MM-DD") : undefined,
+                    occurredBefore: period !== "ALL" ? endDate.format("YYYY-MM-DD") : undefined,
+                    updatedAfter: lastUpdated ? moment(lastUpdated).format("YYYY-MM-DD") : undefined,
                     fields: { $all: true },
                 })
                 .getData();
         };
 
-        const result = await promiseMap(orgUnits, async orgUnit => {
-            const { events, pager } = await fetchApi(orgUnit, 1);
+        const result: D2TrackerEvent[][] = await promiseMap(orgUnits, async orgUnit => {
+            const { instances, total = 0, pageSize } = await fetchApi(orgUnit, 1);
 
-            const paginatedEvents = await promiseMap(_.range(2, pager.pageCount + 1), async page => {
-                const { events } = await fetchApi(orgUnit, page);
-                return events;
+            const pageCount = Math.ceil(total / pageSize);
+
+            const paginatedEvents = await promiseMap(_.range(2, pageCount + 1), async page => {
+                const { instances } = await fetchApi(orgUnit, page);
+                return instances;
             });
 
-            return [...events, ..._.flatten(paginatedEvents)];
+            return [...instances, ..._.flatten(paginatedEvents)];
         });
 
         return _(result)
             .flatten()
-            .filter(({ programStage }) => programStageIds.includes(programStage))
+            .filter(({ programStage }) => programStageIds.includes(programStage || ""))
```

Comment:
Is programStage `""`  within the domain of programStageIds? If not, I'd write: `(programStage ? programStageIds.includes(programStage) : false))`